source/GlassView.Export.Test/DirectoryExportTest.cs

```diff
@@ -0,0 +1,45 @@
+using System.Text.Json;
+using BenchmarkDotNet.Loggers;
+using Atmoos.GlassView.Core;
+using Atmoos.GlassView.Core.Models;
+
+namespace Atmoos.GlassView.Export.Test;
+
+public class DirectoryExportTest
+{
+    private static readonly FileInfo testFile = new(Path.Combine("Resources", "TestBenchmark.json"));
+
+    [Fact]
+    public async Task ExportWritesJsonFile()
+    {
+        var options = new JsonSerializerOptions().EnableGlassView();
+        var summary = testFile.Deserialize<BenchmarkSummary>(options);
+        var emptyDirectory = Directory.CreateDirectory(Guid.NewGuid().ToString());
```

Comment:
We shouldn't create directories and files in our test classes. See the concept behind [wrap static classes to improve testability](https://www.codementor.io/@copperstarconsulting/improve-testability-by-wrapping-static-classes-zfd45xl65).
This is also what my first contribution to Atmosphere is going to be.
With this concept, we would no longer depend on Directory or File.

Reply:
Unfortunately, there is already a more advanced nuget out there, so I will no longer commit that.
Here's the Repo we could be referencing
https://github.com/TestableIO/System.IO.Abstractions

Reply:
I agree with the premisse, but feel it's not a show stopper for this PR. (If you agree please resolve this comment)
Let's take this discussion to the general level like discussing editorconfig etc.

I've created issue #14